client/src/components/ChatPage/ChatPage.tsx

```diff
@@ -3,13 +3,25 @@ import { ChatList } from '../ChatList/ChatList';
 import { Chat } from '../Chat/Chat';
 import { useCallback, useState } from 'react';
 import { NewChat } from '../NewChat/NewChat';
+import { WelcomeChatWindow } from '../WelcomeChatWindow/WelcomeChatWindow.tsx';
+import { Conversation } from '../../services/conversation-service.ts';
 
 export function ChatPage() {
   const [isNewChatPending, setIsNewChatPending] = useState(false);
   const onCreateNewClick = useCallback(() => {
     setIsNewChatPending(!isNewChatPending);
+    setSelectedConversation(undefined);
   }, [isNewChatPending]);
 
+  const [selectedConversation, setSelectedConversation] = useState<
```

Comment:
useState < Conversation > () is enough, react understands that could be undefined in this case

Reply:
If we plan to call  get /conversations/:id to fetch the full chat with messages, we can keep only the `conversation.id` here and then pass that around instead of the full Conversation object
so the above would be useState< string >()